_data/sidebars/community_sidebar.yaml

```
@@ -11,17 +11,69 @@ entries:
     url: /community.html
     output: web
 
-  - title: Coupled Problems 2025
-    url: /eccomas-coupled-2025.html
+  - title: preCICE workshops
     output: web
+    folderitems:
 
-  - title: preCICE Workshop 2024
-    url: /precice-workshop-2024.html
-    output: web
-  
-  - title: iacm WCCM 2024
-    url: /iacm-wccm-2024.html
+    - title: The preCICE Workshop
+      url: /precice-workshop.html
+      output: web
+
+    - title: Workshop 2024
+      url: /precice-workshop-2024.html
+      output: web
+
+    - title: Workshop 2023
+      url: /precice-workshop-2023.html
+      output: web
+
+    - title: Workshop 2022
+      url: /precice-workshop-2022.html
+      output: web
+
+    - title: Workshop 2021
+      url: /precice-workshop-2021.html
+      output: web
+
+    - title: Workshop 2020
+      url: /precice-workshop-2020.html
+      output: web
+
+  - title: preCICE sessions
```

```suggestion
  - title: preCICE minisymposia
```
?
It is a common term in academia. "session" might be too vague?